Assets/Scripts/Navigation/Map.cs

```
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Map : MonoBehaviour {
+
+	public GameObject hexPrefab;
+
+	int width = 20;
+	int height = 20;
+
+	float xOffset = 0.96f;
+	float yOffset = 0.71f;
+
+	public string mapShape = "hex";
+
+	public void Start(){
+		BuildMap();
+	}
+
+	void Update(){
+
+	}
+
+	private void BuildMap(){
+		switch(mapShape.ToUpper()){
+			//Builds a hex shaped map
+			case "HEX"	: //Todo
```

Instead of a Todo, just have it call the HexMap() method, where it should throw an error